test.sh

```diff
@@ -36,7 +36,7 @@ do
 done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
 rm -f .libs/tests-linux64.*
-rock --allerrors -q -lpthread --gc=off $ARGS $FLAGS $TESTS_USE_FILE && ./Tests
+rock -q -lpthread --gc=off $ARGS $FLAGS $TESTS_USE_FILE && ./Tests
```

Comment:
That is actually why I went for not displaying all errors. It is not that help full then.